core/backup/fx.js

```
@@ -0,0 +1,901 @@
+                const data = {};
+                for (const key in instance) {
+                    if (Object.prototype.hasOwnProperty.call(instance, key) && typeof instance[key] !== 'function') {
+                        data[key] = instance[key];
+                    }
+                }
+                output.__value = JSON.stringify(data);
+            } else if (this.__type === 'html' && this.__value.dom && IS_CLIENT) {
+                const el = this.__value.dom;
+                output.__value = {
+                    tagName: el.tagName,
+                    attributes: Array.from(el.attributes).reduce((acc, attr) => ({ ...acc, [attr.name]: attr.value }), {}),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)